src/installer/tests/Assets/TestProjects/HostApiInvokerApp/HostFXR.cs

```diff
@@ -23,6 +24,45 @@ internal enum hostfxr_resolve_sdk2_result_key_t : int
                 global_json_path = 1,
             }
 
+            [StructLayout(LayoutKind.Sequential)]
+            internal struct hostfxr_dotnet_environment_sdk_info
+            {
+                internal int size;
+                [MarshalAs(UnmanagedType.LPWStr)]
```

Comment:
This is wrong - you should not need to add any marshalling info I think.
* The `char_t` is defined as UTF-16 on windows, and UTF-8 everywhere else
* Default marshalling should work that way as well